docs/concepts/functions-overview.md

```diff
@@ -38,10 +38,20 @@ common function methods:
 * set_env(name, value)
 * set_envs(env_vars)
 * gpus(gpus, gpu_type)
-* with_limits(mem, cpu, gpus, gpu_type)
-* with_requests(mem, cpu)
 * set_env_from_secret(name, secret, secret_key)
 
+The limits methods are different for Spark and Dask:
+- Spark
+   - with_driver_limits(mem, cpu, gpu_type)
+   - with_executor_limits(mem, cpu, gpu_type)
+   - with_driver_resources(mem, cpu, gpu_type)
+   - with_executor_resources(mem, cpu, gpu_type)
```

Comment:
`with_*_resources` is not a thing. remove please

Reply:
removed